### PR TITLE
Add formatting scripts

### PR DESCRIPTION
**Pull request by ajchili**
## What?

This PR adds formatting to our package via `prettier`.

## Why?

Consistent formatting across our files is important for consistency and maintainability. Rather than requiring each individual to manually do this, we can automated. This is automated via `prettier` and validated at build time.

### CODE REVIEW

**Comment by ajchili** [comment]
This PR does not format files because then it would be so hard to parse. Will do another PR for that.

**Review by KentTDang — APPROVED** [review]
This looks good to push :)
